lib/dry/auto_inject/strategies/args.rb

```diff
@@ -10,9 +12,9 @@ class Args < Constructor
         def define_new
           class_mod.class_exec(container, dependency_map) do |container, dependency_map|
             define_method :new do |*args|
-              deps = dependency_map.to_h.values.map.with_index { |identifier, i|
+              deps = dependency_map.to_h.values.map.with_index do |identifier, i|
                 args[i] || container[identifier]
-              }
+              end
```

Comment:
My personal preference is to use the curly braces when the return value of the method accepting the block is significant, which is why I've done that here. I'd rather we didn't change it. In fact, these days I usually just disable the Rubocop rule about this.